src/app/workspace/project/project-settings.component.ts

```diff
@@ -269,4 +275,36 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy {
         )
       );
   }
+
+  public uploadRawProjectContent(file: File) {
+    combineLatest([
+      this.store$.pipe(select(selectOrganizationByWorkspace)),
+      this.store$.pipe(select(selectProjectByWorkspace)),
```

Comment:
There is selector called `selectWorkspaceModels` which selects organization & project.